src/main/java/net/sf/jabref/gui/preftabs/ExternalTab.java

```diff
@@ -88,7 +88,7 @@ public ExternalTab(JabRefFrame frame, PreferencesDialog prefsDiag, JabRefPrefere
         JLabel commandDescription = new JLabel(
                 "<html>" +
                         Localization.lang("<u>Note</u>: Use the placeholder <i>%0</i>" +
-                                " for the location of the current database file.", "%DIR") +
```

Comment:
I liked "current" more than "opened": There is always one "current" database file (or zero if nothing is opened :)), but possibly more than one opened.